index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML Quiz</title>
+  <link rel="stylesheet" type="text/css" href="main.css">
+
+</head>
+
+<body>
+
+  <div class="container">
+    <div class="column">
+      <header>
+        <h1>Green Yoda</h1>
+      </header>
+      <article>
+        <section>
+          <p><img src="img/yodaGreen.png"></p>
+          <p class="justifyText">Yoda's voice recognition makes him the most realistic Star Wars toy ever! He speaks 115 phrases that lead to Jedi Wisdom and Lightsaber mastery.
+            Yoda's realistic movements display unmatched Jedi Lightsaber skills; watch Yoda perform his signature 360 Lightsaber attack!
+            Legendary Yoda is for Jedi's aged 6+. 6 C batteries required. Batteries not included.
+            Includes: 1 Legendary Yoda, 1 Lightsaber, 1 Cane
+          </p>
+          <p class="greybar">Availability</p>
+        </section>
+        <div class="text-pad">
```

Comment:
Can you do this without adding an extra div?  Lets keep all of the `<section/>`'s at the same level.